source/math/FloatMatrix.ooc

```diff
@@ -45,6 +45,36 @@ FloatMatrix : cover {
 		result
 	}
 
+	setColumn: func (x: Int, values: ...) {
```

Comment:
It should be possible to use `data height` `memcpy` calls instead of a loop and multiple operator function calls. Another thing is that memory access will be better if you switch the order of the loops (`for each row` x `for each column` is more efficient for a row-major data, as it is more likely that the continuous row data is on the same memory page).
These are minor details. Good idea, though.